tests/framework_cherrypy/tox.ini

```diff
@@ -0,0 +1,29 @@
+[tox]
+setupdir = {toxinidir}/../..
+envlist =
+    {py35,py36,py37,py38,py39,pypy3}-CherryPy14,
+    {py35,py36,py37}-CherryPy0302,
+    pypy3-CherryPy0303
```

Comment:
Should we maybe add the latest version of CherryPy to this matrix? Possibly just replacing CherryPy14 if there's no difference in coverage?